src/templates_front/templator_front.org

```diff
@@ -559,15 +560,16 @@ typedef int32_t back_end_t;
 #define TREXIO_DELIM            "\n"
    #+end_src
 
-   The helper function ~trexio_has_backend~ returns ~true~ if TREXIO compilation includes a back end provided as an argument; ~false~ otherwise.
+   The helper function ~trexio_has_back_end~ returns ~true~ if TREXIO compilation includes a back end provided as an argument; ~false~ otherwise.
    This is useful due to the fact that HDF5 back end can be disabled at configure step.
 
    #+begin_src c :tangle prefix_front.h
 bool trexio_has_backend(back_end_t back_end);
+bool trexio_has_back_end(back_end_t back_end);
    #+end_src
 
    #+begin_src c :tangle prefix_front.c
-bool trexio_has_backend(back_end_t back_end) {
+bool trexio_has_back_end(back_end_t back_end) {
```

Comment:
My €0.02: I find "backend" easier on the eye than "back_end" (neither is incorrect, especially as variable name), and it would take about as many changes to change "back_end" to "backend" as it does to do it the other way around, with the advantage that no legacy function aliases are required if I am not mistaken (constant aliases are needed though).

Reply:
OK, both are fine for me. The only important thing is to have a unque way to write it :-)

Reply:
I propose that we accept this PR, and I will submit another one with `backend` instead of `back_end`.

Reply:
Sure, you can count me as "approved" - all that's left is Evgeny's request for un-kinded `integer`s (which I agree with).

Reply:
Yes, everything is fine with me as long as we can converge on the `integer`-s mangling in `src/templates_front/build.sh`.